src/app/space/wizard/space-wizard.module.ts

```diff
@@ -12,22 +13,16 @@ import { SpaceWizardComponent } from './space-wizard.component';
 
 @NgModule({
   imports: [
+    AddSpaceOverlayModule,
```

Comment:
Don't need to import AddSpaceOverlayModule. We're only broadcasting in space-wizard component.

Reply:
removed